cmd/siac/main.go

```diff
@@ -124,6 +125,7 @@ func main() {
 	renterAllowanceCmd.AddCommand(renterAllowanceCancelCmd)
 
 	renterCmd.Flags().BoolVarP(&renterListVerbose, "verbose", "v", false, "Show additional file info such as redundancy")
+	renterContractsCmd.Flags().BoolVarP(&renterAllContracts, "all", "A", false, "Show all expired contracts in addition to active contracts")
```

Comment:
is there a reason that we use captial A and capital H as the flags instead of the lowercase alternatives?

Reply:
lower case a is a global flag i think so I did capital A to avoid any potential conflicts. If we prefer lower case letters I can do `old` and `o` instead.